cdap-cli-tests/src/test/java/co/cask/cdap/cli/CLIMainTest.java

```diff
@@ -523,6 +523,9 @@ public void testWorkflowToken() throws Exception {
     testCommandOutputContains(
       cli, String.format("get workflow token %s %s at node %s scope user key %s", workflow, runId,
                          FakeWorkflow.FakeAction.ANOTHER_FAKE_NAME, FakeWorkflow.FakeAction.TOKEN_KEY), fakeNodeValue);
+
+    // stop workflow
+    testCommandOutputContains(cli, "stop workflow " + workflow, "400: Program not running");
```

Comment:
However, I don't think that needs to be tested here, as it's more of an integration test for workflows. What's being tested here is whether the CLI is hitting the correct HTTP endpoint.